package.json

```diff
@@ -65,6 +65,8 @@
     "aws-sdk": "^2.140.0",
     "aws4": "^1.8.0",
     "axios": "^0.18.0",
+    "lodash": "^4.17.10",
+    "moment": "^2.22.2",
```

Comment:
serverless in version 1.26.0 has transitive dependency: https-proxy-agent in version 1.0.0
github is reporting:
"Known high severity security vulnerability detected in https-proxy-agent < 2.2.0 defined in package-lock.json.
--
package-lock.json update suggested: https-proxy-agent ~> 2.2.0."